manifests/config.pp

```diff
@@ -20,13 +23,20 @@
     order   => '01',
   }
 
-  file { [$pulpcore::user_home, $pulpcore::webserver_static_dir, $pulpcore::cache_dir]:
+  file { $pulpcore::user_home:
     ensure => directory,
     owner  => $pulpcore::user,
     group  => $pulpcore::group,
     mode   => '0775',
   }
 
+  file { [$pulpcore::cache_dir, $pulpcore::chunked_upload_dir, $pulpcore::media_root]:
+    ensure => directory,
+    owner  => $pulpcore::user,
+    group  => $pulpcore::group,
+    mode   => '0770',
```

Comment:
`README.md` states that the `media_root` directory has `0750` permissions. Does that need to be split off here, or updated in the `README.md` to `0770` instead?

Reply:
Good point. I think 750 should be sufficient and it was a copy paste error from the home dir (I did fix 5 -> 0, forgot about the group). Will update.